hack/tools/pkg/janitor/janitor_test.go

```diff
@@ -348,36 +328,30 @@ func Test_janitor_CleanupVSphere(t *testing.T) {
 	// Initialize and start vcsim
 	clients, sim := setup(ctx, t)
 
-	deleteAll := time.Now().Add(time.Hour * 1)
-
 	tests := []struct {
 		name               string
 		dryRun             bool
-		maxCreationDate    time.Time
 		objects            []*vcsimObject
 		wantAfterFirstRun  map[string]bool
 		wantAfterSecondRun map[string]bool
```

Comment:
Having these two does not make sense anymore and the tests should fail currently :think:

Reply:
(only one of them should be needed)

Reply:
Not sure. There is a case where it needs to runs to also cleanup folders/resourcePools. Do we have to change our implementation?

Reply:
Turns out the tests are not run in CI. Maybe because govc is not setup there? (or because of hack/tools)